c_api/include/cpp_demangle.h

```diff
@@ -8,6 +8,8 @@ extern "C" {
 struct DemangleOptions {
   // Do not display function arguments.
   bool no_params;
+  // Do not display the function return type.
+  bool no_return_type;
```

Comment:
I think the naming is fine. A single function has multiple parameters but only one return type, so this makes sense.